cpython-unix/build.py

```diff
@@ -817,6 +801,11 @@ def build_cpython(
         if "lto" in parsed_build_options:
             env["CPYTHON_LTO"] = "1"
 
+        sdk_path = os.environ["APPLE_SDK_PATH"]
+        if not os.path.exists(sdk_path):
+            raise Exception("macOS SDK path %s does not exist" % sdk_path)
+        env["APPLE_SDK_PATH"] = sdk_path
```

Comment:
This check is already performed around line 170. Do we need to check this again?

Reply:
Yeah will DRY this up once it's working as expected.